crates/aiken-lang/src/parser/token.rs

```diff
@@ -183,6 +183,6 @@ impl fmt::Display for Token {
             Token::Validator => "validator",
             Token::Via => "via",
         };
-        write!(f, "\"{s}\"")
+        write!(f, "{s}")
```

Comment:
I am not 100% sure about the impact of this one. And also not sure why we were wrapping tokens in double-quotes when printing them. This was ugly in the error reporting so I removed them; but I am not certain that we don't show them in other places where the quotes might matter.

Anyway, we should add the quotes in-place where they're needed, and not from this function.